### ISSUE TITLE
Also boot emulator/Simulator

### ISSUE THREAD

**Issue by jfversluis** [tuyen-vuduc/maui-runner#1]
Hey, thanks for this awesome initiative! Do you have any future plans with it? I think it would make it even much more useful if you could also boot the emulator or Simulator with this for instance :)

**Comment by tuyen-vuduc** [comment]
@jfversluis Thank you.

Yeah, that's a good idea to also boot the emulator or Simulator.

I used to think about it. However, there are quite a lot of iOS simulators, showing them all on CLI won't look good.

I am thinking about a VSCode extension will serve better or an add-in to C# tool.

**Comment by jfversluis** [comment]
Yeah that's all pretty cool ideas! I definitely like the VSCode extension, I think a lot of people will like that. Is something like that included in Comet already as well? You might be able to get some inspiration from that too

**Comment by tuyen-vuduc** [comment]
Yeah, I didn't managed to run Comet on my machine yet. I will check it out again.

**Comment by jfversluis** [comment]
Looks like this could be handy as well: https://github.com/redth/AndroidSdk.Tools

**Comment by tuyen-vuduc** [comment]
It looks cool. Thanks for sharing, @jfversluis .
